internal/pkg/tfconf/output.go

```diff
@@ -2,8 +2,9 @@ package tfconf
 
 // Output represents a Terraform output.
 type Output struct {
-	Name        string   `json:"name"`
-	Description String   `json:"description"`
+	Name        string `json:"name"`
+	Description String `json:"description"`
+	Value       string
```

Comment:
Have to change to:
```
Value       string   `json:"value,omitempty" yaml:"value,omitempty"`
```
So that json and yaml will include the attribute if it isn't empty and so that the capitalisation of the attribute matches the others in json/yaml.

Reply:
Yep, you have to add the yaml tag